migrate.py

```diff
@@ -0,0 +1,77 @@
+import os
+import psycopg2
+from config import dbconfig, filename, section
+
+
+
+def create_tables():
+    """create tables in postgresql database"""
+    commands = (
+        """
+        DROP TABLE IF EXISTS tb_users;
+        CREATE TABLE tb_users(
+            user_id SERIAL PRIMARY KEY,
+            username VARCHAR(100) UNIQUE NOT NULL,
+            firstname VARCHAR(50) NOT NULL,
+            lastname VARCHAR(50) NOT NULL,
+            password VARCHAR(255) NOT NULL,
+            created_on TIMESTAMP NOT NULL,
+            last_login TIMESTAMP
```

Comment:
You should add a field to store the user role ie, `admin` or `normal user`

Reply:
Ok, I'll do that